src/System Application/App/AI/src/Copilot/CopilotCapabilityImpl.Codeunit.al

```diff
@@ -278,6 +278,10 @@ codeunit 7774 "Copilot Capability Impl"
         CopilotCapability: Enum "Copilot Capability";
     begin
         CopilotCapability := Enum::"Copilot Capability".FromInteger(Capability);
+
+        if CopilotCapability in [Enum::"Copilot Capability"::Chat, Enum::"Copilot Capability"::Summarize] then
+            Silent := true;
+
```

Comment:
Why?
The `silent` parameter bubbles up all the way to the top of the stack, so we should just make sure the caller sets it to true instead of hardcoding true here.

Reply:
I assume Summarize is a platform capability, and as such, when you're invoking the event, you can set the caller to `true` like Enrico mentioned. This event should not set it for individual features, because there are circumstances that App's subscriber cannot account for.

Reply:
nice :)